docs/notebooks/running_test_application.ipynb

```diff
@@ -0,0 +1,354 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example of running sar-pre-processing package"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Requirements\n",
+    "\n",
+    "- Installation of sar-pre-processing package\n",
+    "    - Problem: gdal package might not be installed correctly\n",
```

Comment:
I did not have this problem. SNAP was able to use its built-in gdal package. I recommend that you either remove this sentence or state what should be done if this problem occurs.

Reply:
Old Problem. Statement removed